From bdf06cd99ae40337b5380b0d575d1e3bda3f71c2 Mon Sep 17 00:00:00 2001
From: Eneas U de Queiroz <cote2004-github@yahoo.com>
Date: Tue, 19 Jun 2018 08:01:14 -0300
Subject: [PATCH] pimbd: update to current git HEAD

Allow compilation with gcc 7.3.0.

Signed-off-by: Eneas U de Queiroz <cote2004-github@yahoo.com>
(cherry-picked from commit c299f8bcea26669bc93db16f59e12f9272a30f8d)
Signed-off-by: Josef Schlehofer <pepe.schlehofer@gmail.com>

Pull request: https://github.com/openwrt-routing/packages/pull/534
Fixes: https://github.com/openwrt-routing/packages/issues/527
---
 pimbd/Makefile | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/pimbd/Makefile b/pimbd/Makefile
index 9f97107..a091119 100644
--- a/pimbd/Makefile
+++ b/pimbd/Makefile
@@ -1,8 +1,8 @@
 include $(TOPDIR)/rules.mk

 PKG_NAME:=pimbd
-PKG_SOURCE_VERSION:=68f5fc803119e4b33a88b35c096f4d6ac28b6de5
-PKG_VERSION:=2015-08-18-$(PKG_SOURCE_VERSION)
+PKG_SOURCE_VERSION:=dbf4e5913b06e3160f506df15e6a047a403a5f21
+PKG_VERSION:=2018-06-19-$(PKG_SOURCE_VERSION)
 PKG_RELEASE:=1

 PKG_SOURCE_PROTO:=git
@@ -17,7 +17,8 @@ include $(INCLUDE_DIR)/package.mk
 include $(INCLUDE_DIR)/cmake.mk

 # Spammy debug builds for now
-CMAKE_OPTIONS += -DL_LEVEL=7
+CMAKE_OPTIONS += -DL_LEVEL=7 \
+		 -DWITH_LIBUBOX=1

 define Package/pimbd
   SECTION:=net
--
2.24.0
